src/image/makeWcs.cc

```diff
@@ -95,13 +95,19 @@ afwImg::Wcs::Ptr afwImg::makeWcs(
         metadata->set<std::string>("CTYPE1", "RA---TAN");
         metadata->set<std::string>("CTYPE2", "DEC--TAN");
         metadata->set<bool>("TPV_WCS", true);
-
-        for (int i = 1; i <= 3; ++i) {
-            for (int j = (i == 1 ? 5 : 1); j <= 16; ++j) { // note that PV1_{1..4} are legal
+        // PV1_[1-4] are in principle legal, but Swarp reuses them as part of the TPV parameterisation
+        // As a compromise, we remove them iff there is at least one PV2_? card
+        bool sawPV2 = false;               // did we see a PV2_? card?
+        for (int i : { 2, 1 }) {
```

Comment:
Range-based for isn't supported by GCC 4.4, so this will cause Buildbot to choke.